core/app/models/spree/store_credit.rb

```diff
@@ -150,7 +150,7 @@ def can_void?(payment)
   end
 
   def generate_authorization_code
-    "#{id}-SC-#{Time.current.utc.strftime('%Y%m%d%H%M%S%6N')}"
+    "#{id}-SC-#{Time.current.utc.strftime('%Y%m%d%H%M%S%6N')}-#{SecureRandom.uuid}"
```

Comment:
This has enough segments in it now that it might be nicer to write it more like this, but either way is fine by me.

```suggestion
    [
      id,
      "SC",
      Time.current.utc.strftime('%Y%m%d%H%M%S%6N'),
      SecureRandom.uuid
    ].join("-")
```